changelogs/current.yaml

```diff
@@ -94,6 +94,12 @@ minor_behavior_changes:
     Disables recvmmsg (multi-message) for reading packets from a client QUIC UDP socket, if GRO
     is not set or not supported. recvmsg will be used instead. This behavior change can be
     reverted by setting ``envoy.reloadable_features.disallow_quic_client_udp_mmsg`` to ``false``.
+- area: lua
+  change: |
+    Previously, tracing spans generated by ``httpCall`` were always sampled by default.
+    Now, the default sampling decision is inherited from the parent span, so that no "orphaned"
+    sampled span associated with the ``httpCall`` is emitted. The default remains overridable
+    with the ``trace_sampled`` option.
```

Comment:
nit: Please point out that we only change the default value in lua:

`... Now, the default sampling decision in lua is .....`

Reply:
How's this?